allocations/admin.py

```diff
@@ -11,7 +11,16 @@
 class ChargeInline(admin.TabularInline):
     model = Charge
     extra = 1
-    fields = ["region_name", "user"]
+    fields = [
+        "region_name",
+        "user",
+        "resource_type",
+        "resource_id",
+        "hourly_cost",
+        "start_time",
+        "end_time",
+    ]
+    readonly_fields = ["region_name", "user"]
```

Comment:
This updated admin page just helps with debugging/checking on charges.

User really needs to be readonly. Otherwise django loads a 10,000+ item dropdown for each charge with all users.